client/src/components/Overview/CartForm.jsx

```diff
@@ -1,16 +1,44 @@
-import React, { useContext } from 'react';
-import ProductContext from '../../context';
+import React, { useState } from 'react';
+
+export default function CartForm({ currentStyle }) {
+  const [selectedSku, setSelectedSku] = useState('default');
+  const [quantity, setQuantity] = useState([]);
```

Comment:
I think it would be clearer to use a plural noun for an array.   E.g. quantities here.  Or just store the number in the state variable "quantity" and iterate elsewhere.

Reply:
will change to the clearer variable name.